invertedai/large/common.py

```diff
@@ -0,0 +1,153 @@
+from typing import List, Optional, Tuple
+from pydantic import BaseModel
+
+from invertedai.common import AgentAttributes, AgentState, RecurrentState, Point
+
+class Region(BaseModel):
+    """
+    A data structure to contain information about a region in a map to be used in large simulations.
+
+    See Also
+    --------
+    AgentAttributes
+    """
+
+    center: Optional[Point] = None # The center of the region if such a concept is relevant (e.g. center of a square, center of a rectangle)
```

Comment:
```suggestion
    center: Optional[Point] = None #: The center of the region if such a concept is relevant (e.g. center of a square, center of a rectangle)
```
Use `#:` for each field to include those comments in the documentation.

Reply:
Done.